integ/failure-server/teardown.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# delete everything in the right order and ignore errors
```

Comment:
```suggestion
# delete everything in the right order. errors are ignored so that this
# script can be used whether or not all of the artifacts exist.
```